pkg/logicalbackup/logicalbackup.go

```diff
@@ -749,30 +652,30 @@ func (b *LogicalBackup) prepareTablesForPublication(conn *pgx.Conn) error {
 
 // returns the LSN from where we should restart reads from the slot,
 // InvalidLSN if no state file exists and error otherwise.
-func (b *LogicalBackup) readRestartLSN() (dbutils.Lsn, error) {
+func (b *LogicalBackup) readRestartLSN() (dbutils.LSN, error) {
 	var stateInfo StateInfo
 	stateFilename := path.Join(b.baseDir(), stateFile)
 
 	if _, err := os.Stat(stateFilename); os.IsNotExist(err) {
-		return dbutils.InvalidLsn, nil
+		return dbutils.InvalidLSN, nil
 	}
 
 	fp, err := os.OpenFile(stateFilename, os.O_RDONLY, os.ModePerm)
 	if err != nil {
-		return dbutils.InvalidLsn, fmt.Errorf("could not create current lsn file: %v", err)
+		return dbutils.InvalidLSN, fmt.Errorf("could not create current lsn file: %v", err)
 	}
 	defer fp.Close()
 
 	if err := yaml.NewDecoder(fp).Decode(&stateInfo); err != nil {
-		return dbutils.InvalidLsn, fmt.Errorf("could not decode state info yaml: %v", err)
+		return dbutils.InvalidLSN, fmt.Errorf("could not decode state info yaml: %v", err)
 	}
 
 	currentLSN, err := pgx.ParseLSN(stateInfo.CurrentLSN)
 	if err != nil {
-		return dbutils.InvalidLsn, fmt.Errorf("could not parse %q Lsn string: %v", stateInfo.CurrentLSN, err)
+		return dbutils.InvalidLSN, fmt.Errorf("could not parse %q LSN string: %v", stateInfo.CurrentLSN, err)
```

Comment:
I thought the idea was to use lowercase in the user-visible messages, but uppercase in the code (certainly this is not consistent with usage of lsn in other error messages).

Reply:
well, then we have change them the other way around in other messages (we can do it as a separate PR not to delay merging this one).